doc/source/getting_started/installing.rst

```diff
@@ -0,0 +1,458 @@
+.. _installing_odl:
+
+##############
+Installing ODL
+##############
+
+************
+Introduction
+************
+
+Installing ODL is intended to be straightforward, and this guide is meant for new users.
+For a minimal installation you should perform the following steps:
+
+1. Install a Python interpreter
+2. Install ODL and its dependencies
+3. (optional) Run the tests
+
```

Comment:
Here we should be more clear that if the reader already has a python installation that they are happy with, they can skip the "install python" steps and go straight to the install ODL step